symbols/overlay01.yml

```diff
@@ -53,6 +53,12 @@ overlay1:
         
         r0: Action ID
         r1: True if the option should be enabled, false otherwise
+    - name: ProcessContinueScreenContents
+      address:
+        EU: 0x23336D8
+      description: |-
+        Fetches the required data and creates all the strings to display the contents shown in the window
+        when choosing continue in the main menu.
```

Comment:
No parameters or return type? Also, this doesn't have a corresponding entry on overlay01.h
Is it because the signature is unknown?